src/Microsoft.DotNet.Build.Tasks.net45/CreateExternalAssembliesResources.cs

```diff
@@ -0,0 +1,78 @@
+// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+// See the LICENSE file in the project root for more information.
+
+using Microsoft.Build.Framework;
+using Microsoft.Build.Utilities;
+using System;
+using System.Collections;
+using System.IO;
+using System.Reflection;
+using System.Resources;
+
+namespace Microsoft.DotNet.Build.Tasks
+{
+    public class CreateExternalAssembliesResources : Task
+    {
+        [Required]
+        public ITaskItem[] InputAssemblies { get; set; }
+
+        [Required]
+        public string OutputPath { get; set; }
+
+        public override bool Execute()
+        {
+            try
+            {
+                if (!Directory.Exists(OutputPath))
+                {
+                    Directory.CreateDirectory(OutputPath);
+                }
+
+                CreateReswFiles();
+            }
+            catch (Exception e)
+            {
+                Log.LogErrorFromException(e, showStackTrace: false);
+            }
+
+            return !Log.HasLoggedErrors;
+        }
+
+        public void CreateReswFiles()
```

Comment:
@weshaggard, as I mentioned extracting the resources was done internally inside GenerateResource task when building ResW. I don't think we can call GenerateResource just to extract the resources

http://index/?query=Microsoft.AppxPackage.Targets&rightProject=CBA(Repo)&file=src%5Ctools%5CBuildProcess%5CMSBuild%5CMicrosoft%5CVisualStudio%5Cv14.0%5CAppxPackage%5CMicrosoft.AppxPackage.Targets&line=1047